portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1144,6 +1147,7 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, geneset_ids,
 		if (data_by_id[geneset_id][case_id].profile_data === null) {
 		    // set the initial value for this sample or patient
 		    data_by_id[geneset_id][case_id].profile_data = sample_score;
+		    data_by_id[geneset_id][case_id].na = false;
```

Comment:
Never mind, I see no evidence of `delete` being inefficient. @adamabeshouse, do you think deleting the property is better than setting it to false, even though it's a boolean?